.editorconfig

```diff
@@ -0,0 +1,5 @@
+# top-most EditorConfig file
+root = true
+
+[*.{cs,json,txt,yml}]
+end_of_line = lf
```

Comment:
So usually I prefer for my repo to have only `auto` but we will keep it like that, I don't mind here.